charts/streams-app/templates/deployment.yaml

```diff
@@ -116,6 +116,16 @@ spec:
             - name: {{ printf "STREAMS_%s" $key | replace "." "_" | upper | quote }}
               value: {{ $value | quote }}
           {{- end }}
+          {{- range .Values.ports }}
+            {{- if .servicePort }} # TODO verify that there is at most one service port. Currently, if there are multiple service ports, the first one will be used
+            - name: POD_IP
+              valueFrom:
+                fieldRef:
+                  fieldPath: status.podIP
+            - name: STREAMS_APPLICATION_SERVER
+              value: "$(POD_IP):{{ .servicePort }}"
```

Comment:
We have to use `containerPort` here because this is used to establish connections from pod to pod

Reply:
Updated